validate.lic

```diff
@@ -510,6 +510,13 @@ class DRYamlValidator
     warn('***YOU HAVE OUTDATED SETTINGS*** THE SETTING lockpick_buffs IS NO LONGER USED. Please make a waggle_set called \'pick\'')
   end
 
+  def assert_that_sell_loot_money_on_hand_is_correct(settings)
+    settings.sell_loot_money_on_hand =~ /(\d+) (\w+)/
+    return if Regexp.last_match(1) && %w[copper silver gold platinum].any? { |denom| denom[Regexp.last_match(2)] }
```

Comment:
What about `bronze`?

Reply:
Does this handle partial matches (e.g. `3 silv` instead of `3 silver`)?

Reply:
Missed bronze, thanks!

Reply:
Yes, Ive tested it on `3 silver`, `3 silv` which pass and `3 basdf` which fails. 
```
2.0.0-p647 :001 > 'silver'['sil']
 => "sil"
```

Reply:
Other things that fail correctly are only including a number because it fails the regex match on the first line

Reply:
We may want to check if the substring is from the beginning of the string.. as in 'silver'['i'] will be a positive match but incorrect denomination.

Reply:
👍